english/explore-outside/journey-to-flaming-tower/walk/portal/wishes/eagle/wishes.md

```diff
@@ -0,0 +1,3 @@
+With the wave of his hands, the little fellow summons an eagle! You both jump on the back of it and 
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.